cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -293,27 +297,38 @@ metadata:
     operator: rook
     storage-backend: ceph
 rules:
-  - apiGroups:
-      - ""
+  - apiGroups: [""]
+    resources: ["secrets", "configmaps"]
     verbs:
-      - "*"
-    resources:
-      - secrets
-      - configmaps
-  - apiGroups:
-      - storage.k8s.io
-    resources:
-      - storageclasses
+      # OBC controller creates secrets and configmaps containing information for users about how to
+      # connect to object buckets. It deletes them when an OBC is deleted.
+      - get
+      - create
+      - delete
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["storageclasses"]
     verbs:
+      # OBC controller gets parameters from the OBC's storageclass
+      # Rook gets additional parameters from the OBC's storageclass
       - get
+  - apiGroups: ["objectbucket.io"]
+    resources: ["objectbucketclaims"]
+    verbs:
+      # OBC controller needs to list/watch OBCs and get latest version of a reconciled OBC
       - list
       - watch
-  - apiGroups:
-      - "objectbucket.io"
+      - get
+      # lib-bucket-provisioner updates the OBC with bucket information (not just the status)
+      - update
```

Comment:
lib bucket provision also creates and deletes obc, right?

Reply:
I don't believe so. Users create and delete OBCs.